### ISSUE TITLE
CoordinatorLayout unmeasured issue with focal point

### ISSUE THREAD

**Issue by cammace** [mapbox/mapbox-gl-native#7940]
When my location tracking is enabled, and the map view is in a coordinator layout, the focal point is set to 0.0 (instead of being null?). This causes strange gesture behavior:

![ezgif com-video-to-gif 3](https://cloud.githubusercontent.com/assets/5652865/22609896/f14d0194-ea30-11e6-87c2-91573682c44d.gif)

cc: @tobrun @zugaldia 

SimpleMapActivity in testapp:
```java
@Override
  public void onMapReady(MapboxMap mapboxMap) {

    mapboxMap.getTrackingSettings().setMyLocationTrackingMode(MyLocationTracking.TRACKING_FOLLOW);
  }
```

Layout:
```xml
<android.support.design.widget.CoordinatorLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    xmlns:mapbox="http://schemas.android.com/apk/res-auto"
    android:orientation="vertical"
    tools:context=".activity.maplayout.SimpleMapActivity">

    <com.mapbox.mapboxsdk.maps.MapView
        android:id="@id/mapView"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        mapbox:mapbox_uiAttribution="false"
        mapbox:mapbox_cameraTargetLat="40.73581"
        mapbox:mapbox_cameraTargetLng="-73.99155"
        mapbox:mapbox_uiCompassMarginTop="32dp"
        mapbox:mapbox_uiLogo="false"
        mapbox:mapbox_styleUrl="mapbox://styles/mapbox/traffic-day-v1"
        mapbox:mapbox_cameraZoom="11"/>

</android.support.design.widget.CoordinatorLayout>
```

**Comment by zugaldia** [comment]
cc: @Guardiola31337 for extra 👀

**Comment by tobrun** [comment]
the underlying issue is that CoordinatorLayout works differently than other viewgroups under the hood. It doesn't have a height and width at the same time as other viewgroups would have. Need to debug this in more detail to suggest a fix.

**Comment by tobrun** [comment]
This will be fixed with picking up #6999:

![ezgif com-video-to-gif 14](https://cloud.githubusercontent.com/assets/2151639/23169155/f4e26a7e-f84a-11e6-9bab-d1a1bd782ee8.gif)

**Comment by tobrun** [comment]
Fixed with https://github.com/mapbox/mapbox-gl-native/issues/6999
